os_installer2/pages/partitioning.py

```diff
@@ -103,7 +107,14 @@ def __init__(self):
         self.column3.add_attribute(ren, "markup", INDEX_PARTITION_PATH)
         self.treeview.append_column(self.column3)
 
-        self.column4 = Gtk.TreeViewColumn("Current filesystem", ren)
```

Comment:
So now we don't know the old filesystem..?

Reply:
I just changed the label to "Filesystem" as it can now be changed from ext4 to xfs. Still loads with whatever filesystem already exists (as far as I can tell, at least).